dashboard/test/ui/features/SectionActionDropdown.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Using the SectionActionDropdown
+  Scenario: Viewing progress from SectionActionDropdown
+    Given I create a teacher-associated student named "Sally"
+    And I give user "Teacher_Sally" hidden script access
+    And I complete the level on "http://studio.code.org/s/allthethings/stage/2/puzzle/1"
+    And I complete the free response on "http://studio.code.org/s/allthethings/stage/27/puzzle/1"
+    And I submit the assessment on "http://studio.code.org/s/allthethings/stage/33/puzzle/1"
+    And I sign out
+
+    When I sign in as "Teacher_Sally"
+    And I am on "http://code.org/teacher-dashboard?no_home_redirect=1"
+    And I click selector "div.title:contains('Student Accounts and Progress')" once I see it
+    And I click selector ".fa-chevron-down" once I see it
+    And I click selector "a:contains('View Progress')" once I see it
+    And I wait for 4 seconds
+    And I wait until current URL contains "/progress"
+
+  Scenario: Managing students from SectionActionDropdown
+    Given I am a teacher
+    And I am on "http://studio.code.org/"
+    And I create a new section
+    And I click selector ".fa-chevron-down" once I see it
+    And I click selector "a:contains('Manage Students')" once I see it
+    And I wait until current URL contains "/manage"
+
+  Scenario: Printing Login Cards from SectionActionDropdown
+    Given I am a teacher
+    And I am on "http://studio.code.org/"
+    And I create a new section
+    And I click selector ".fa-chevron-down" once I see it
+    And I click selector "a:contains('Print Login Cards')" once I see it
+    And I wait until current URL contains "/print_signin_cards"
+
+  Scenario: Editing Section Information from SectionActionDropdown
+    Given I am a teacher
+    And I am on "http://studio.code.org/"
+    And I create a new section
+    And I click selector ".fa-chevron-down" once I see it
+    And I click selector "#ui-test-edit-section" once I see it
+    And I press the save button to create a new section
```

Comment:
This doesn't read correctly.  We're not actually creating a new section at this point in the test, right?  Instead, what is the condition we are trying to assert?  Let's write a new step for that if it doesn't exist.

Reply:
Would it be appropriate to re-name this step 'And I press the save button to save this section'? The point of that statement is to assert that the edit section dialog showed up (if it's not available, we can't press this button). Renaming it would still work in the original context (as part of creating a new section) and not mislead here.

Reply:
Or even just "And I press the save button" would be fine.  The implementation of that step is:
```rb
When /^I press the save button to create a new section$/ do
  steps 'When I press the first ".uitest-saveButton" element'
end
```
So it will press any button with the `.uitest-saveButton` class.